secrets/vault/vault.go

```diff
@@ -18,13 +18,12 @@
 //
 // URLs
 //
-// For secrets.OpenKeeper URLs, vault registers for the scheme "vault"; URLs
-// start with "vault://". secrets.OpenKeeper will dial a Vault server once per
-// unique combination of the following supported URL parameters:
-//   - address: Sets Config.APIConfig.Address; should be a full URL with the
-//       address of the Vault server.
-//   - token: Sets Config.Token; the access token the Vault client will use.
-// Example URL: "vault://mykey?address=http://vault.server.com:8080&token=aaaaa".
+// For secrets.OpenKeeper, vault registers for the scheme "vault".
+// The default URL opener will dial a Vault server using the environment
+// variables "VAULT_SERVER_URL" and "VAULT_SERVER_TOKEN".
+// To customize the URL opener, or for more details on the URL format,
+// see URLOpener.
+// See https://godoc.org/gocloud.dev#URLs for background information.
```

Comment:
s/`https://godoc.org/gocloud.dev#URLs`/`https://godoc.org/gocloud.dev#hdr-URLs`

And also for comments in other files.

Reply:
Nice catch, done. Also fixing the As links.